scripts/release.sh

```diff
@@ -9,7 +9,7 @@ function release {
     echo "Building and releasing $tag..."
     echo "Pushing docker image..."
     echo ${DOCKER_PASS} | docker login --username ${DOCKER_USER} --password-stdin
-    docker tag ${DOCKER_REPO}:latest
+    docker tag ${DOCKER_REPO} ${DOCKER_REPO}:latest
```

Comment:
hmm, does this work, I feel like we might need:

```shell
    docker tag ${DOCKER_REPO}:${tag#v} ${DOCKER_REPO}:latest
```

Reply:
I think you're right actually, closing #958 in favor of #959